docs/keys_and_certificates.md

```diff
@@ -302,3 +329,91 @@ keyUsage               = critical, keyCertSign, cRLSign
 subjectKeyIdentifier   = hash
 authorityKeyIdentifier = keyid:always
 ```
+
+### Putting the chain together
+
+PEM files contain certificates (among other objects that are not relevant here)
+encoded as text. They can be concatenated e.g. with `cat` to form a chain. As
+[mentioned above](#informational-checks), the order of certificates matters:
+leaf (i.e. PO issuing certificate) must come first, root CA - last. Assuming
+`*.crt` files are PEM certificates of various CAs, full chain is produced by:
+
+```shell
+cat root.crt intermediate.crt issuer.crt > chain.pem
+```
+
+### TL;DR version
```

Comment:
This is a great solution; I would appreciate it if we would use consistent naming across the whole testing process (including options of attester and provisioner apps) and link this section in relevant places in other documentation (at this point I know about building, but there may be others).

Reply:
Changed and added link in https://github.com/fobnail/docs/commit/ee9239f5f3f4e4d6d42c21c0cd2f9bf2d615f076